app/stac_api/serializers.py

```diff
@@ -514,23 +515,37 @@ class Meta:
 
     # NOTE: when explicitely declaring fields, we need to add the validation as for the field
     # in model !
+    item = serializers.SlugRelatedField(
+        slug_field='name', write_only=True, queryset=Item.objects.all()
+    )
+    id = serializers.CharField(
+        source='name',
+        required=True,
+        max_length=255,
+        validators=[validate_name, UniqueValidator(queryset=Asset.objects.all())]
+    )
+    title = serializers.CharField(max_length=255, required=False, allow_blank=True)
```

Comment:
I would not add the `title` explicitly here, it is not needed. Note that when adding explicitly fields we need to make sure that the validation is done as in the model ! See comments.

Reply:
see above